src/histolab/filters/image_filters.py

```diff
@@ -169,6 +193,15 @@ class RgbToGrayscale(ImageFilter):
     -------
     PIL.Image.Image
         Grayscale image
+
+
+
+    Example:
+        >>> from PIL import Image
+        >>> from histolab.filters.image_filters import RgbToGrayscale
+        >>> image_rgb = Image.open("/path/to/image.png") # Read an RGB image
```

Comment:
Here i can see a problem that we could have with running doctests, `"/path/to/image.png"` is a fake path obviously but this will break the doctest runner

Reply:
yup... alllll in your hands :)